lib/helper/Polly.js

```diff
@@ -134,8 +160,20 @@ class Polly extends Helper {
     return handler.intercept((_, res) => res.send(data));
   }
 
+  _mockRequestForWebDriver(method, oneOrMoreUrls, dataOrStatusCode, additionalData) {
+    const webDriverIOConfigUrl = this.helpers.WebDriver && this.helpers.WebDriver.options.url;
+    this.browser.execute(
```

Comment:
this looks like async operation, so probably should be made an async function or at least return a promise